permissions/plugin-oak9.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: "oak9"
+github: &GH "jenkinsci/oak9-plugin"
```

Comment:
Is this format ok?

Reply:
yes it's a yaml anchor

Reply:
It's not used in this case, but it won't hurt anything.